settingtypes.txt

```diff
@@ -30,11 +32,14 @@ postool.hud.defaultshowblock (Show block coordinates by default) bool true
 postool.hud.defaultshowmesecons (Show mesecon info by default) bool false
 # Show mesecon details by default (requires mesecons_debug
 postool.hud.defaultshowmeseconsdetails (Show mesecon details by default) bool false
+# Show biome data by default
+postool.hud.defaultshowbiome (Show biome data by default) bool false
 # wait at least this long before updating HUD (seconds)
 postool.hud.minupdateinterval (Update HUD interval) int 2 1 999
 # wait at least this long before grid vanishes
 postool.tool.griddisplayduration (Grid display duration) int 12 1 999
 # don't show chunk border indicator. Only shows up when not in center block.
 # Setting this to true makes it never show at all.
 postool.tool.suppresschunkindicator (Hide chunk border indicator) bool false
-
+# Show the heat and humidity values of your position to two decimal places.
+postool.biomedata.verbose (More biome data) bool false
```

Comment:
This is part of the HUD, the setting name should reflect that: postool.hud.biomedata.verbose or similar and as such should be placed up where it's relative settings are.